components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -864,6 +864,26 @@ void BraveWalletService::MigrateUserAssetsAddIsNFT(PrefService* prefs) {
   prefs->SetBoolean(kBraveWalletUserAssetsAddIsNFTMigrated, true);
 }
 
+// static
+void BraveWalletService::MigrateHiddenNetworks(PrefService* prefs) {
+  auto previous_version_code =
+      prefs->GetInteger(kBraveWalletDefaultHiddenNetworksVersion);
+  if (previous_version_code < 1) {
```

Comment:
Return early `if (previous_version_code >= 1)` (latest version that nothing to migrate), `prefs->SetInteger(kBraveWalletDefaultHiddenNetworksVersion, 1);` shouldn't need to be run if this is true.